src/ariel-os-embassy/src/delegate.rs

```diff
@@ -41,10 +43,9 @@ pub struct Delegate<T> {
     send: Signal<CriticalSectionRawMutex, SameExecutorCell<*mut T>>,
     reply: Signal<CriticalSectionRawMutex, ()>,
     was_exercised: AtomicBool,
+    _not_send: PhantomData<*const ()>,
```

Comment:
This actually breaks the usb_builder_hook use case. why did we add the `!Send` bound, can we just drop it instead?

Reply:
ah yes, thanks! ... I totally misread the compilation error. I've added the explicit `Sync` impl.